terraform/README.md

```diff
@@ -183,8 +215,50 @@ follow these instructions. This setup needs to be run only once per AWS account.
 - Create an s3 bucket. Add the bucket's name to `main.tf` > `terraform` > `backend` > `bucket`
 - Create a [DynamoDB](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/getting-started-step-1.html) table
   called `terraform-lock` with `LockID` as primary key.
-- Create a directory in this repository similar to `development/` with the name of the chosen AWS profile set in its `main.tf`.
-  See [managing environments](#managing-environments) to customize your infrastructure.
+
+### 2. Add the new environment to the repository
+
+Run the following script from the `terraform/` directory to create a directory for your new environment. If your new
+environment's name is `kerala`:
```

Comment:
Let's use a generic name like `sample`?

Reply:
Oh I actually think a specific name is much more helpful. It's a great way to indicate what that thing is semantically - is it usually a country? An env (like production), etc.

Reply:
Ah, makes sense 👍🏼  An `india` or `bangladesh` would be more recognisable here since they're countries.